yikes, docs with python 3.8 

```
Run make -C docs html
make: Entering directory '/home/runner/work/datalad-extension-template/datalad-extension-template/docs'
python -m sphinx -b html -d build/doctrees  -W source build/html
Running Sphinx v7.[1](https://github.com/datalad/datalad-extension-template/actions/runs/15085297131/job/42407077667?pr=98#step:6:1).2
Traceback (most recent call last):
  File "../../setup.py", line 7, in <module>
    from _datalad_buildsupport.setup import (
  File "/home/runner/work/datalad-extension-template/datalad-extension-template/_datalad_buildsupport/setup.py", line 18, in <module>
    from setuptools.errors import OptionError
```

Let's boost Python there from EOLed 3.8

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -41,7 +41,7 @@
 #     will run until the file is removed (or 60 min have passed)
 #
 #   - in a terminal execute, for example, `C:\datalad_debug.bat 39` to set up the
-#     environment to debug in a Python 3.8 session (should generally match the
+#     environment to debug in a Python 3.9 session (should generally match the
 #     respective CI run configuration).
 
 
@@ -80,7 +80,7 @@ environment:
     - ID: MacP38core
       DTS: datalad_helloworld
       APPVEYOR_BUILD_WORKER_IMAGE: macos-monterey
-      PY: 3.8
+      PY: 3.9
       INSTALL_GITANNEX: git-annex
       DATALAD_LOCATIONS_SOCKETS: /Users/appveyor/DLTMP/sockets
       CODECOV_BINARY: https://uploader.codecov.io/latest/macos/codecov

--- a/.github/workflows/docbuild.yml
+++ b/.github/workflows/docbuild.yml
@@ -13,10 +13,10 @@ jobs:
         git config --global user.email "test@github.land"
         git config --global user.name "GitHub Almighty"
     - uses: actions/checkout@v4
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
       uses: actions/setup-python@v5
       with:
-        python-version: 3.8
+        python-version: 3.9
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip

--- a/.github/workflows/docbuild.yml
+++ b/.github/workflows/docbuild.yml
@@ -19,7 +19,7 @@ jobs:
         python-version: 3.9
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
+        python -m pip install --upgrade pip setuptools
         pip install -r requirements-devel.txt
         pip install .
     - name: Build docs

--- a/.github/workflows/test_crippledfs.yml
+++ b/.github/workflows/test_crippledfs.yml
@@ -28,10 +28,10 @@ jobs:
         git config --global user.email "test@github.land"
         git config --global user.name "GitHub Almighty"
     - uses: actions/checkout@v4
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
       uses: actions/setup-python@v5
       with:
-        python-version: 3.8
+        python-version: 3.9
     - name: Install dependencies
       run: |
         pip install -r requirements-devel.txt

--- a/changelog.d/pr-98.md
+++ b/changelog.d/pr-98.md
@@ -1,0 +1,5 @@
+### 🐛 Bug Fixes
+
+- BF: use setuptools.errors.OptionError instead of now removed import of distutils.DistutilsOptionError.
+  Stop using python 3.8 in CI (use 3.9).
+  [PR #98](https://github.com/datalad/datalad-extension-template/pull/98) (by [@yarikoptic](https://github.com/yarikoptic))

